src/lib.ts

```diff
@@ -448,6 +450,20 @@ export function userConfirmDialog(msg: string, defaultValue: boolean, timeOut?:
   return userConfirm(msg);
 }
 
+export const latteActionSourceFinderConstraints = {
+  allowedAction: (action: ActionSource): boolean => {
+    if (!have($item`latte lovers member's mug`)) return true;
+    const forceEquipsOtherThanLatte = (
+      action?.constraints?.equipmentRequirements?.().maximizeOptions.forceEquip ?? []
+    ).filter((equipment) => equipment !== $item`latte lovers member's mug`);
+    return (
+      forceEquipsOtherThanLatte.every((equipment) => toSlot(equipment) !== $slot`off-hand`) &&
+      forceEquipsOtherThanLatte.filter((equipment) => toSlot(equipment) === $slot`weapon`).length <
```

Comment:
Possible that garbo does 2 or 3 handed items?

Reply:
very good point. What I should do is something where I sum the weaponHands of all the forceEquipsOtherThanLatte